src/components/Dashboard/Dashboard.js

```diff
@@ -460,15 +460,17 @@ class Dashboard extends React.Component {
               ref={(ref) => (this.responseRef = ref)}
               optionsToolbarRef={this.optionsToolbarRef}
             />
-            <OptionsToolbar
-              authentication={getAuthentication(this.props.authentication)}
-              autoQLConfig={getAutoQLConfig(this.props.autoQLConfig)}
-              themeConfig={getThemeConfig(this.props.themeConfig)}
-              onErrorCallback={this.props.onErrorCallback}
-              onSuccessAlert={this.props.onSuccessCallback}
-              ref={(r) => (this.optionsToolbarRef = r)}
-              responseRef={this.responseRef}
-            />
+            <div style={{ display: 'none' }}>
```

Comment:
When I solved ANT-647, I added `<OptionsToolbar/> `in the drilldown table. But it caused this new bug AC-939. Then I just hide the `<OptionsToolbar/>` in the drilldown table but we can still access the` report problem `function.